modules/tinymce/src/plugins/template/test/ts/browser/DialogGetPreviewContentTest.ts

```diff
@@ -10,54 +10,55 @@ import { getPreviewContent } from 'tinymce/plugins/template/ui/Dialog';
 import { Settings } from '../module/Settings';
 
 const metaKey = Env.os.isMacOS() || Env.os.isiOS() ? 'e.metaKey' : 'e.ctrlKey && !e.altKey';
+const host = document.location.host;
```

Comment:
This is something we've known about for quite sometime and is logged as TINY-6575, so you might want to link this to that.

Reply:
I linked the tickets will make sure that the previous gets closed as well. Added details about that one in the description as well.